### ISSUE TITLE
No release tag for 0.2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bestlong/krank#4]
```
What steps will reproduce the problem?

1. Try to do a tag of the source repo:
svn --username chrismathiaster --password ***** copy
"https://krank.googlecode.com/svn/code/trunk/validation"
https://krank.googlecode.com/svn/code/tags/Release 0.2 with CRUD" -m
"Tagging the 0.2 release of crank"

svn: DELETE of '/svn/!svn/act/f8c89450-ab6b-7a4f-b3d9-c8365bac7999': 500
Internal Server Error (https://krank.googlecode.com)

so basically our 0.2 release is NOT tagged - FYI.

```

Original issue reported on code.google.com by `chrismat...@gmail.com` on 24 Apr 2007 at 6:07
